### ISSUE TITLE
Commandbar items not working responsive browsers

### ISSUE THREAD

**Issue by hrc7505** [OfficeDev/office-ui-fabric-react#1238]
Hi,

We are trying to use your component Commandbar . We just need top menu option from that and when we reduce size of the browser menu options should go in More Options (... Three Dots).

But we are not able to achieve that.

Here we have some code snippet that we implemented.

```
import * as React from 'react';
import { assign } from 'office-ui-fabric-react/lib/Utilities';
import { CommandBar } from 'office-ui-fabric-react/lib/CommandBar';

export class Demo extends React.Component<any, any> {

  constructor(props: any) {
    super(props);
  }

  public render() {
    let  items  = [
            {name:"Menu1"},
            {name:"Menu2"},
            {name:"Menu3"},
            {name:"Menu4"},
            {name:"Menu5"},
            {name:"Menu6"},
            {name:"Menu7"},
            {name:"Menu8"},
            {name:"Menu9"},
            {name:"Menu10"}
           ];
           
    let filteredItems = items.map(item => assign({}, item, {
      name:  item.name
    }));
    
    return (
      <div>
        <CommandBar
          elipisisAriaLabel='More options'
          items={ filteredItems }
        />
      </div>
    );
  }
} 
```

We have attached[ output video URL](https://drive.google.com/open?id=0B6hbZQAGDxuKNy1ET1hnRHFhaGc) of output of this code.

Let us know if that is a bug or we are doing something wrong.

Thanks,

**Comment by dzearing** [comment]
Strange. I am investigating. We'll get this fixed shortly.

Here's a repro with latest bits:
http://codepen.io/dzearing/pen/ENjKPy

In Chrome: I see the ellipsis menu showing up but the measurements seem off, as in it's not collapsing soon enough.

In Safari: it's not working for some reason. Will see what can be done.

**Comment by hrc7505** [comment]
Hey @dzearing 

Thanks for the helping us. Yes we were able to fix it up and now we have 3 dots as more options.

issue was in array of Menu Items we were not passing "key" value to it. So when we added key to each object of that array it started working.

I am still wondering why we need to put "Key"there. And if we don`t put a "Key" there, than shouldn`t it take "name" as "Key"of that.

Anyways its working now, Thanks for the help.
